chaos/chaos.go

```diff
@@ -8,20 +8,28 @@ package main
 
 import (
 	"fmt"
-	"github.com/miekg/dns"
 	"net"
 	"os"
+	"sync"
+
+	"github.com/miekg/dns"
 )
 
 func main() {
 	if len(os.Args) != 2 {
 		fmt.Printf("%s NAMESERVER\n", os.Args[0])
 		os.Exit(1)
 	}
-	conf, _ := dns.ClientConfigFromFile("/etc/resolv.conf")
+	conf, err := dns.ClientConfigFromFile("/etc/resolv.conf")
+	if err != nil {
+		fmt.Println("error making client from default file", err)
```

Comment:
Your call really, but the rest of the program is using fmt.Prin* and then os.Exit